js/_website/src/lib/components/Demos.svelte

```diff
@@ -26,33 +41,11 @@
 				Open in 🎢 ↗
 			</button>
 
-			<gradio-lite playground shared-worker class="p-2">
+			<gradio-lite playground shared-worker layout="vertical" class="p-2">
```

Comment:
Annoying suggestion, but can the demo go before the code? Users now have to scroll down a lot to see the demo.

![playground_lite_docs](https://github.com/gradio-app/gradio/assets/41651716/559128dd-accd-475e-b89d-eb1a7bf409ff)

Reply:
Good point that's kind of annoying. I'll fix it on the lite playground side and not directly on the website. Opened an issue https://github.com/gradio-app/gradio/issues/8228